### PR TITLE
fix(container-runtime): only send OpRoundtripTime for ops, not noops

### PR DESCRIPTION
**Pull request by WayneFerrao**
[AB#7583](https://dev.azure.com/fluidframework/235294da-091d-4c29-84fc-cdfc3d90890b/_workitems/edit/7583)

## Description

This PR investigates and fixes a bug in which some OpRoundtripTime events are sent without any performance data (`durationInboundToProcessing`,  `durationOutbound` and `durationNetwork`). This happened because the events were being sent on noop messages as well as regular ops.

This resulted in gaps in the Op Latency view which causes gaps in the graph. 

This fix also removes the need to check for ops with the performance data.

### CODE REVIEW

**Comment by alexvy86** [comment]
Did we get to the bottom of _why_ can this happen? I think that will inform if the fix is correct or not :). If this is something that might occur naturally because of X and Y well-understood reasons, it might be ok, but we really need to understand what those reasons are.

File: packages/runtime/container-runtime/src/connectionTelemetry.ts
Comment (seanimam) [inline packages/runtime/container-runtime/src/connectionTelemetry.ts:364]: Great find. Before we merge this, we need to determine if there is a value to emitting the noop version of this message If there is, Maybe the change we make instead is to ignore it within devtools. And if not, perhaps we should include the message type here?

Comment (seanimam) [reply packages/runtime/container-runtime/src/connectionTelemetry.ts:364]: Also need to confirm, it is a bug for noops to be missing statistics. If so maybe we can make a change here so that they do.

**Review by seanimam — APPROVED** [review]

**Pull request merged**
